pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -44,7 +44,7 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/pkg/scheduler/algorithm"
 	priorityutil "k8s.io/kubernetes/pkg/scheduler/algorithm/priorities/util"
-	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
+	legacyapi "k8s.io/kubernetes/pkg/scheduler/api"
```

Comment:
I intentionally alias it to "legacyapi", which will get resolved in a followup PR.